sdk/servicebus/Azure.Messaging.ServiceBus/CHANGELOG.md

```diff
@@ -2,24 +2,33 @@
 
 ## 7.18.0-beta.2 (Unreleased)
 
+### Acknowledgments
+Thank you to our developer community members who helped to make the Service Bus client library better with their contributions to this release:
+
+- Martin Costello _([GitHub](https://github.com/martincostello))_
+
 ### Features Added
 
 ### Breaking Changes
 
 ### Bugs Fixed
 
-- Fixed an error that caused connection strings using host names without a scheme to fail parsing and be considered invalid.
+- Fixed an issue that caused connection strings using host names without a scheme to fail parsing and be considered invalid.
+
 - Fixed an issue where the scheduled enqueue time was not cleared when creating a new message from a received message.
 
-- Fixed an error that prevented relative URIs from being used with [application properties](https://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-application-properties) in the `ServiceBusMessage.ApplicationProperties` and `ServiceBusReceivedMessage.ApplicationProperties` collections. 
+- Fixed an issue that prevented relative URIs from being used with [application properties](https://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-application-properties) in the `ServiceBusMessage.ApplicationProperties` and `ServiceBusReceivedMessage.ApplicationProperties` collections. 
+
+- Fixed an issue that caused `ServiceBusMessageBatch` to accept more than the allowed 1mb batch limit when sending to Service Bus entities with large message sizes enabled.
 
 ### Other Changes
 
 - The client will now refresh the maximum message size each time a new AMQP link is opened; this is necessary for large message support, where the maximum message size for entities can be reconfigureed adjusted on the fly.  Because the client had cached the value, it would not be aware of the change and would enforce the wrong size for batch creation.
 
 - Updated the `Microsoft.Azure.Amqp` dependency to 2.6.7, which contains a fix for decoding messages with a null format code as the body.
 
-- Instances of `ServiceBusSender` created with no explicit `ServiceBusSenderOptions` value allocate less memory.
+- Improved efficiency of subclient creation, reducing allocations when no explicit options are passed.  - Fixed deserialization of the lock token to take into account endianness. _(A community contribution, courtesy of [martincostello](https://github.com/martincostello))_
```

Comment:
@jsquire might be an issue because I look at this on the phone but the fixed de serialization bullet looks misplaced

Reply:
dang it.  That's a real thing!   Thanks for the heads-up, Daniel.